frontend/src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
When searching for "html doctype", *all* examples I could find use the uppercase version. I don't really see a reason to change this: it might confuse other devs but doesn't give us any advantage apart from being less screamy.

Reply:
For completeness' sake: [The standard](https://html.spec.whatwg.org/multipage/syntax.html#the-doctype) says it's fine, but even **it** uses the uppercase consistently, so yeah okay, not gonna fight you on this.

Just let the record state I don't like it! :stuck_out_tongue_closed_eyes: